src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -493,6 +493,22 @@ public async Task<FhirResponse<Bundle>> PostBundleAsync(Resource bundle, Cancell
             return await CreateResponseAsync<Bundle>(response);
         }
 
+        public async Task<FhirResponse<Bundle>> PostBundleAsyncWithValidationHeader(Resource bundle, string profileValidation, CancellationToken cancellationToken = default)
```

Comment:
Just curious - are we not intending to expose this via the interface IFhirClient?

Reply:
Thank you. This will enable client consumers to access this method using the interface rather than the actual underlying impl obj.

Reply:
nit: this should be `PostBundleWithValidationHeaderAsync`